uitools/cpp/Esri/ArcGISRuntime/Toolkit/BasemapGalleryController.cpp

```diff
@@ -0,0 +1,689 @@
+/*******************************************************************************
+ *  Copyright 2012-2021 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+#include "BasemapGalleryController.h"
+
+// Toolkit headers
+#include "Internal/GeoViews.h"
+
+// ArcGISRuntime headers
+#include <Map.h>
+#include <Scene.h>
```

Comment:
From my perspective, angle brackets are the right call. Quotes are for searching for "files under control of the implementation," while angle brackets are for files not under control of the current implementation. See: https://en.cppreference.com/w/cpp/preprocessor/include

From a toolkit perspective, the toollkit is an SDK consumer that is part of the source code of the user's implementation (and not a standalone library itself), so I think this makes more sense. 

I would even go as far to say that our samples, as consumers of the SDK, should also switch to angle brackets, being SDK consumers as opposed to implementors.

Reply:
You _could_ make the tenuous claim that angle brackets will lead to a faster compile, because the search doesn't start from the current directory of the source file. But I doubt that holds water.